tests/features/api_datarequest.feature

```diff
@@ -56,6 +123,20 @@ Feature: Datarequest API
         And the Yoda datarequest assignment get API is queried with request id
         Then the response status code is "200"
 
+    Scenario: Datarequest is reviewer
```

Comment:
A condition should be added that specifies that the user is assigned as a reviewer. E.g. `And "<user>" is assigned as reviewer of datarequest`.

Reply:
This is handled in 'the result is "<result>"'?